### PR TITLE
Assessments v1 apis

### PR DESCRIPTION
**Pull request by ZombieWoofers**
- GET/PATCH for assessments
- singular `assessment` copy for existing POST
- include assessments[] with attendanceForSession call

### CODE REVIEW

File: src/main/mule/coach.xml [hunk 1]
```diff
@@ -662,11 +732,25 @@ output application/json
 		<logger
 			level="INFO"
 			doc:name="Log Stored URI Params"
-			doc:id="f5fb5455-22c7-4d1f-b316-fa26806f3f1b"
+			doc:id="5776800c-73ac-4089-a7f5-963b32492dbc"
 			message="URI Param - coachId: #[vars.coachId], teamseasonId: #[vars.teamSeasonId], sessionId: #[vars.sessionId] - stored as vars" />
+  
+
+
+  <flow-ref doc:name="secondaryFlow" name="get:\sessions\(sessionId)\assessments:salesforce-data-api-config" target="sessionAssessments"/>
```
Comment (arch-jn) [inline src/main/mule/coach.xml:740]: What's this flow for? @ZombieWoofers

**Review by arch-jn — CHANGES_REQUESTED** [review]

**Comment by arch-jn** [comment]
@CritiqalPete @ZombieWoofers I thought we were planning to move away from the coach endpoints and use more generic endpoints that are not tied to coachId. Are the new coach endpoints being added intentionally?

File: src/main/mule/coach.xml (hunk 1 above)
Comment (ZombieWoofers) [reply src/main/mule/coach.xml:740]: @archana-llamas this is the approach we used to include session assessment data in the attendance response, we originally tried within the salesforce query but weren't successful.

**Comment by ZombieWoofers** [comment]
@archana-llamas @CritiqalPete  I wasn't aware of that with coachId, I'm fine to adjust this PR as well.

**Comment by CritiqalPete** [comment]
Good catch. I concur.

**Comment by CritiqalPete** [comment]
@ZombieWoofers did you remove the CoachID dependency? Looks like the PR needs updating.

**Comment by CritiqalPete** [comment]
this is no longer needed and has been implemented successfully as POST /coach/(coachId)/teamseasons/(teamSeasonId)/sessions/(sessionId)/assessments

the recommendation this be at an endpoint besides /coach has been appreciated.

**Pull request closed without merging**